ExploreASL_Initialize.m

```diff
@@ -600,6 +600,33 @@ function xASL_init_AddDirsOfxASL(MyPath)
                 end
             end
         
+			% Check and remove empty spaces at the start and end of the DatasetRoot
+			if ~isempty(x.opts.DatasetRoot)
+
+				% Remove the empty spaces at the start
+				if x.opts.DatasetRoot(1) == ' '
+					[~, ~, indexUniqueChars] = unique(x.opts.DatasetRoot, 'stable'); % Create a vector that gives for each character its unique identifier
```

Comment:
This gives the rank right? "unique identifier" is very vague

When ranking 1000 2000 1500, you get 1 3 2, this is what you mean right?

Reply:
No. A unique unsorted identified. 
1000 2000 1500 gives 1,2,3

Reply:
I think you can simplify this using regular expressions that search for spaces. But fine with this implementation.

Just a minor note: if you want to explain what is happening, it is not only nice to explain what your code does technically (e.g., aacba gives 1 1 2 3 1) but also why you do this (e.g.,

% Remove empty spaces at the start of the path
% First, we check if the first index is a space
% Second, we check the position of the second unique character (the first unique character being a space).
% This position is where the path starts (there can be multiple spaces that need to be omitted)

Reply:
Good idea. Actually, for PLDs, I had to write this code, which I then ported here. But you are right that with regexp, it is much easier, so I redid all that now with regexp.

The explanations is now also much more simple

Reply:
Hmm you say that you redid everything with regexp, but that's not what I see in the code?

Reply:
Sorry. Forgot to push indeed.